lib/devices/runtime_arg.ml

```diff
@@ -32,6 +32,8 @@ let runtime_network_key ~pos fmt =
 
 let delay = runtime_arg ~pos:__POS__ "delay"
 
+let name = runtime_arg ~pos:__POS__ "name_k"
```

Comment:
I think if you use this you potentially register `name_k` twice and I don't know what happens if you do that.